File: spring-boot-cxf-swagger/src/main/java/com/chen/demo/service/UserServiceImpl.java
package com.chen.demo.service;

import com.chen.demo.entity.User;
import io.swagger.annotations.ApiModel;
import org.springframework.stereotype.Component;

import javax.ws.rs.core.Response;

@Component

public class UserServiceImpl implements UserService {
    @Override
    public Response getAccessToken(String credentials, String password, String validation, int isRememberAccount) {
        Response.ResponseBuilder responseBuilder = Response.status(Response.Status.OK);
        return responseBuilder.build();
    }

    @Override
    public Response updateAccessToken(User user) {


        return Response.ok(user).build();
    }
}

File: sprng-jcahce/src/main/kotlin/com/chen/demo/sprngjcahce/service/UserServiceImpl.kt
package com.chen.demo.sprngjcahce.service

import com.chen.demo.sprngjcahce.entity.User
import org.springframework.cache.annotation.Cacheable
import org.springframework.stereotype.Service

@Service
class UserServiceImpl : UserService {
    @Cacheable(cacheNames = ["users"])
    override fun getUser(): User {
        var user = User("chen", 3, "chen-xin-yuan")

        println("------------create user in the userService -------")
        return user
    }


}